setup.cfg

```diff
@@ -8,4 +8,6 @@ universal = 1
 test=pytest
 
 [tool:pytest]
-addopts = --ignore falcon --ignore examples
+filterwarnings =
+    ignore:Unknown REQUEST_METHOD. '(CONNECT|DELETE|GET|HEAD|OPTIONS|PATCH|POST|PUT|TRACE|CHECKIN|CHECKOUT|COPY|LOCK|MKCOL|MOVE|PROPFIND|PROPPATCH|REPORT|UNCHECKIN|UNLOCK|UPDATE|VERSION-CONTROL)':wsgiref.validate.WSGIWarning
```

Comment:
As it happens, pytest does its own warnings collection and so our `warnings.filterwarnings()` call doesn't help in this case.